### ISSUE TITLE
[BUG] KirbyHeaderActions: Buttons from header actions are not hidden in toolbar

### ISSUE THREAD

**Issue by TTengs** [kirbydesign/designsystem#3708]
- [x] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the bug
When trying to hide buttons from a header actions group, the buttons from the page-content are hidden but are now shown in the toolbar instead. If the page is reloaded they are hidden as intended, or if the action group is initiated as not shown.

## Describe how to reproduce the bug
Using the following Stackblitz the buttons can be seen at all times, and the wished behaviour is to to be like the flag:
https://stackblitz.com/edit/stackblitz-starters-m9rrcs?file=src%2Fmain.ts

## Which Kirby version was used?
@kirbydesign/designsystem: 9.6.0

## What was the expected behavior?
The buttons were expected the behave like the flags.

## Please complete the following information:
- OS: Windows
- Browser: Chrome
- Browser version: 130.0.6723.117

## Are there any additional context?
Current fix is to hide the actions with styling like in this example:
https://stackblitz.com/edit/stackblitz-starters-sk4m3t?file=src%2Fmain.ts

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#the-process-of-contributing) correctly.

### Verification
To make sure the bug is not intended behaviour; it should be verified by a member of team Kirby before moving on to implementation. 

- [ ] Make sure the `NOT verified` label has been removed by a member of [team Kirby](https://github.com/kirbydesign/designsystem/blob/develop/.github/SUPPORT.md#team-kirby); do not proceed until this is done.

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Create a test that reproduces the bug following guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)". 
- [ ] Write code to fix the bug

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request.

**Comment by RasmusKjeldgaard** [comment]
Closing this issue automatically because it has been open for more than 6 months and is not currently in progress. If this is still relevant, please feel free to reopen it or open a new issue with updated details. Thank you!
